kazoo/version.py

```diff
@@ -1 +1 @@
-__version__ = '2.8.0'
```

Comment:
FYI black, in my PR, will complain about the single quotes.

On a separate note, I think we should move to setuptools_scm and have this file generated for us from git tags, what do you think?

Reply:
Love the idea!

Reply:
I have it ready to go, I'll submit a PR tomorrow. But I'd like it on top of my black PR if possible